src/Contrib/Dispatcher/Masa.Contrib.Dispatcher.Events/LocalEventBus.cs

```diff
@@ -14,7 +14,7 @@ public class LocalEventBus : ILocalEventBus
     private readonly IUnitOfWork? _unitOfWork;
```

Comment:
Line 36, Remove this argument from the method call; it hides the caller information.

Reply:
Processed